Makefile

```diff
@@ -87,12 +87,12 @@ else
 endif
 
 ifdef ROCM_PATH
-	CFLAGS += -I$(ROCM_PATH)/hip/include
-	LDFLAGS += -L$(ROCM_PATH)/hip/lib
+	CFLAGS += -I$(ROCM_PATH)/hip/include -I$(ROCM_PATH)/hsa/include
```

Comment:
I feel that maybe we only need to do -I$(ROCM_PATH)/include, which works for apt distribution